ddtrace/tracer/tracer.go

```diff
@@ -228,6 +239,15 @@ func newUnstartedTracer(opts ...StartOption) *tracer {
 	if spans != nil {
 		c.spanRules = spans
 	}
+	var dataStreamsProcessor dsminterface.Processor
+	if c.enableDataStreamsMonitoring {
+		if c.agent.DataStreams {
+			log.Info("Enable Data Streams Monitoring")
+			dataStreamsProcessor = datastreams.NewProcessor(c.statsdClient, c.env, c.serviceName, c.agentURL, c.httpClient)
+		} else {
+			log.Info("Can't enable Data Streams Monitoring. Upgrade your agent to 7.34+")
```

Comment:
This should maybe be a `Warn` since they've enabled data streams but it can't be enabled due to the agent version.